packages/lib/src/core/Analytics/events/AnalyticsErrorEvent.ts

```diff
@@ -67,7 +66,7 @@ export class AnalyticsErrorEvent extends AbstractAnalyticsEvent {
         if (props.message) this.message = props.message;
     }
 
-    public getEventCategory(): string {
-        return ANALYTICS_EVENT.error;
+    public getEventCategory(): 'error' {
```

Comment:
Here we could extract the error option from the union type 

```suggestion
    public getEventCategory(): Extract<AnalyticsEventCategory "error"> {
```

Related to this comment https://github.com/Adyen/adyen-web/pull/3675/files#r2564924803